src/proxy-ui-api/frontend/src/views/Clients/InternalServers/InternalServers.vue

```diff
@@ -68,9 +68,7 @@
             v-bind:key="certificate.hash"
           >
             <td class="cert-icon">
```

Comment:
Not changed in this PR but this `td` element has different padding compared to the one below (`pl-4 pt-2`)

Reply:
Fixed